Commands/CLI/promote.js

```diff
@@ -0,0 +1,52 @@
+const { writeJSONAtomic: writeFile } = require("fs-nextra");
+const { CLI: { UpdateConfig } } = require("../../Modules/Utils");
+const configJSON = require("../../Configurations/config.json");
+
+module.exports = async ({ cli }, cmdData, args) => {
+	args = args.trim();
+	const [action, ...ids] = args.split(" ");
+	let isSudo = false;
+	if (ids[0].toLowerCase() === "sudo") {
+		isSudo = true;
+		ids.shift();
+	}
+
+	if (ids.length === 0) {
+		winston.warn(`I need the IDs of the maintainers.`);
+		return;
+	}
+	const isMaintainer = (user, sudo) => sudo ? configJSON.sudoMaintainers.includes(user) : configJSON.maintainers.includes(user);
+	const getMaintainerObject = sudo => sudo ? configJSON.sudoMaintainers : configJSON.maintainers;
+	const add = (user, sudo) => {
+		getMaintainerObject(sudo).push(user);
+	};
+	const remove = (user, sudo) => {
+		getMaintainerObject(sudo).splice(getMaintainerObject(sudo).indexOf(user), 1);
+	};
+	const updateCfg = new Promise(rs => ids.forEach((user, i) => {
+		if (action === "add") {
+			if (isMaintainer(user, isSudo)) {
+				winston.warn(`${user} is a maintainer already.`);
+				return;
+			}
+			if (isSudo && !isMaintainer(user, isSudo)) add(user, isSudo);
+			if (!isMaintainer(user, false)) add(user, false);
+		} else if (action === "remove") {
+			if (!isMaintainer(user, isSudo)) {
+				winston.warn(`${user} is not a maintainer.`);
+				return;
+			}
+			if (isSudo && isMaintainer(user, isSudo)) remove(user, isSudo);
+			if (isMaintainer(user, false)) remove(user, false);
+		}
+		if (i === (ids.length - 1)) return rs();
+	}));
+
+	await updateCfg;
+	try {
+		await UpdateConfig(cli, configJSON);
+		winston.info(`Promoted/demoted users to/from ${isSudo ? "Sudo" : ""} Maintainer`);
```

Comment:
Can't you make the string based on `action`?

Reply:
file size is the key vlag.